common/lib/capa/capa/capa_problem.py

```diff
@@ -496,16 +496,26 @@ def get_answer_ids(self):
         return answer_ids
 
     def find_correct_answer_text(self, answer_id):
-        xml_elems = self.tree.xpath('//*[@id="' + answer_id + '"]')
-        if not xml_elems:
+        """
+        Returns the correct answer(s) for the provided answer_id as a single string.
+
+        Arguments::
+            answer_id (str): a string like "98e6a8e915904d5389821a94e48babcf_13_1"
+
+        Returns:
+            str: A string containing the correct answer or a comma-separated
+                 list of the correct answers.
```

Comment:
I believe it is `comma-separated string` than a `list`?

Reply:
Yup, sorry that was a bit ambiguous.